src/components/ContainerHeader.vue

```diff
@@ -278,6 +271,13 @@ export default {
                 network.ircClient.join(channelName);
             }
         },
+        nicklistToggle: function nicklistToggle() {
+            if (this.sidebarState.sidebarSection === 'user') {
+                this.sidebarState.close();
```

Comment:
If it's currently on the userbox then it go back to the nicklist, not close the whole sidebar